docs/api-docs/storefront/widgets/widgets-tutorial.md

```diff
@@ -115,16 +106,20 @@ Accept: application/json
 

```

Comment:
Since you changed the response, do you need to remove the following:
{
  "name": "Header Images",
  "template": "{{#each images}}<a href='{{image_url}}'><img src={{image_source}} style='width:33.3%'/></a>{{/each}}"
}

Reply:
The widget template did not change. We do not need to remove this example.